ft-contactpage

jeannettuwama opened the pull request:
#### What does this PR do?
[what does this PR do?]
>This PR is about contact where users can feel free to send their queries
#### Description of Task to be completed
[the tasks that you completed on the feature]
>I have completed creating the contact form 
>I have designed the input type with css of course
>I have used JS queries as well 
#### How should this be manually tested?
>clone this repo with https://github.com/jeannettuwama/my_brand/tree/ft-contactpage
>checkout to ft-contactpage


#### What are the relevant Trello story?
>https://trello.com/b/S5RSYwGe/myportfolio
#### Screenshots (if appropriate)

>![ftcont](https://user-images.githubusercontent.com/73231767/98990698-7a478780-2533-11eb-97fe-65e265a22853.PNG)

jeannettuwama committed:
ft-contactpage

mukunzidd commented on UI/JS/contact.js:
```diff
@@ -0,0 +1,30 @@
+function validation(){
+    var name = document.getElementById("name").value;
+    var phone = document.getElementById("phone").value;
+    var email = document.getElementById("email").value;
+    var message = document.getElementById("message").value;
+    var error_message = document.getElementById("error_message");
+    var text;
```
Please use const instead of var for declaring variables.

mukunzidd reviewed (changes requested):
- Your JS file which contains validations is not loading on your page
- Your "submit" button should have the same style as other buttons on your website.
- Your contact form's submit button should have a more specific text like send message or send... just to reflect the action being taken.

mukunzidd commented:
Also, remove the green background and put a light shade of grey color in the background instead of green.

mukunzidd commented:
Also, answer all the questions in the PR description, eg: how should this manually tested.

jeannettuwama committed:
ft (contact page): doing some changing on my feature branch
I have changed the color of background body backgroud from green to grey
Jvascript was not working in my codes due to how the the files were connected but now i have made changes

jeannettuwama committed:
ft (contact page): doing some changing on my feature branch
I have changed the color of background body backgroud from green to grey
Jvascript was not working in my codes due to how the the files were connected but now i have made changes

mukunzidd commented:
What's the update on this @jeannettuwama?

Also from the new screenshot, the form is not spaced correctly. the form elements are squished against the border of the form.

jeannettuwama commented:
I actually did the contact page accoding to the feedback i got from you,
 unless i have to do some more changes,

tjank you

On Tue, 17 Nov 2020 at 18:53, mukunzidd <notifications@github.com> wrote:

> What's the update on this @jeannettuwama
> <https://github.com/jeannettuwama>?
>
> Also from the new screenshot, the form is not spaced correctly. the form
> elements are squished against the border of the form.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jeannettuwama/my_brand/pull/2#issuecomment-729058958>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AROW3F4YHRPRSP6BK2VNQIDSQKS7NANCNFSM4TJI2XJA>
> .
>
